draft-ietf-tls-tls13.md

```diff
@@ -3159,37 +3165,41 @@ The exact source of each of these secrets depends on the operational
 mode (DHE, ECDHE, PSK, etc.) and is summarized in the table below:
 
 ~~~
-    Key Exchange            Static Secret (SS)    Ephemeral Secret (ES)
-    ------------            ------------------    ---------------------
-    (EC)DHE                   Client ephemeral         Client ephemeral
-    (full handshake)       w/ server ephemeral      w/ server ephemeral
+    Key Exchange     Static Secret (SS)    Ephemeral Secret (ES)
+    ------------     ------------------    ---------------------
+    1-RTT (EC)DHE       N/A (SS = ES)        Client Ephemeral
+                                           + Server Ephemeral
 
-    (EC)DHE                   Client ephemeral         Client ephemeral
-    (w/ 0-RTT)                w/ server static      w/ server ephemeral
+    0-RTT (EC)DHE     Client Ephemeral       Client Ephemeral
+                      + Server Static      + Server Ephemeral
 
-    PSK                         Pre-Shared Key           Pre-shared key
+    PSK                Pre-Shared Key          N/A (ES = SS)
 
-    PSK + (EC)DHE               Pre-Shared Key         Client ephemeral
-                                                    w/ server ephemeral
+    PSK + (EC)DHE      Pre-Shared Key        Client Ephemeral
+                                           + Server Ephemeral
 ~~~
 
 These shared secret values are used to generate cryptographic keys as
 shown below.
 
 The derivation process is as follows, where L denotes the length of
-the underlying hash function for HKDF {{RFC5869}}.
+the underlying hash function for HKDF {{RFC5869}} and SS/ES denotes
+the sources from the table above. Whilst SS and ES may be the same
+in some cases, the extracted static_secret and ephemeral_secret
+will not.
 
 ~~~
   HKDF-Expand-Label(Secret, Label, HashValue, Length) =
        HKDF-Expand(Secret, Label + '\0' + HashValue, Length)
 
-  1. xSS = HKDF(0, SS, "extractedSS", L)
+  1. static_secret = HKDF(0, SS, "extracted SS", L)
 
-  2. xES = HKDF(0, ES, "extractedES", L)
+  2. ephemeral_secret = HKDF(0, ES, "extracted ES", L)
```

Comment:
This isn't correct. the static secret is the DH ZZ.

Also, the seeds are a technical change, so remove.